src/FilesFilter.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core;
+
+class FilesFilter
+{
+    protected const FILES = [
```

Comment:
Maybe this `const` should be more readable? `FILES` are a broad concept.

Reply:
Name `FilesFilter` also should be more precise. Or maybe you could just put this as a function in your controller?

Reply:
I think IGNORE FILES is a clearer name